crates/bevy_reflect/src/lib.rs

```diff
@@ -471,6 +472,7 @@ mod impls {
     #[cfg(feature = "smallvec")]
     pub use self::smallvec::*;
     pub use self::std::*;
+    pub use self::uuid::*;
```

Comment:
Nit: Is this needed? In fact, are any of these needed (since I'm pretty sure none of them export anything)?

Reply:
I just followed what I saw

Reply:
That's fine, we can leave it for now.